<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="playlist">
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ping</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>First Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Second Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IP-Address</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Server</string>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>94</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Listen</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFil">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCommand">
    <property name="title">
     <string>Command</string>
    </property>
    <addaction name="actionKick_Player"/>
    <addaction name="actionBan_Address"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Game"/>
    <addaction name="actionSave_Game"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Game"/>
    <addaction name="actionServer_Shutdown"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnectToLocal"/>
    <addaction name="actionConnectToHost"/>
    <addaction name="separator"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuFil"/>
   <addaction name="menuConnection"/>
   <addaction name="menuCommand"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnectToLocal">
   <property name="text">
    <string>ConnectToLocal</string>
   </property>
  </action>
  <action name="actionConnectToHost">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ConnectToHost</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionKick_Player">
   <property name="text">
    <string>Kick Player</string>
   </property>
  </action>
  <action name="actionBan_Address">
   <property name="text">
    <string>Ban Address</string>
   </property>
  </action>
  <action name="actionServer_Shutdown">
   <property name="text">
    <string>Server Shutdown</string>
   </property>
  </action>
  <action name="actionSave_Game">
   <property name="text">
    <string>Save Game</string>
   </property>
  </action>
  <action name="actionStart_Game">
   <property name="text">
    <string>Start Game</string>
   </property>
  </action>
  <action name="actionReset_Game">
   <property name="text">
    <string>Reset Game</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
